frontend/src/metabase/home/components/EmbedHomepage/InteractiveTabContent.tsx

```diff
@@ -55,7 +57,12 @@ export const InteractiveTabContent = ({
         <List.Item>{t`Customize the look and feel of your application to match your brand.`}</List.Item>
       </List>
 
-      <ExternalLink href={interactiveEmbeddingQuickstartUrl}>
+      <ExternalLink
+        href={interactiveEmbeddingQuickstartUrl}
+        // ExternalLink stops clicks events by default
```

Comment:
I didn't want to touch that component which is used in multiple places, so I used this workaround to re-enable clicks